jsk_perception/audio_to_spectrogram/audio_to_spectrogram.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+from cv_bridge import CvBridge
+import numpy as np
+
+import rospy
+
+from audio_common_msgs.msg import AudioData
+from sensor_msgs.msg import Image
+
+
+# This node publish spectrogram (sensor_msgs/Image)
+# from audio (audio_common_msgs/AudioData)
+# Respeaker is assumed to be used.
```

Comment:
Which part depends on respeaker?

Reply:
Thank you for your review.

I added dependency like below:
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2478/files#diff-28ff2534b886febe47348c0b55f459bfR14-R15